docker/Dockerfile

```diff
@@ -0,0 +1,107 @@
+FROM --platform=linux/amd64 node:16.13.0 AS frontend
```

Comment:
As per our discussion, we are going to remove this and allow docker to choose the platform and find a way to fix the issue for M1. have you tried that?

Reply:
removed this platform flag

Reply:
no..